### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 101)

### LINKED ISSUE #2579: [devtools-translate] Russian - What's New in DevTools (Chrome 101)
Post: https://developer.chrome.com/blog/new-in-devtools-101/
Translated by: Apr 20

Translator: @solarrust
Reviewers: @webmaxru, @kateryna-prokopenko

### PR DESCRIPTION
**Pull request by solarrust**
Fixes #2579

Will be happy for review from amazing @kateryna-prokopenko and @webmaxru ♥️

Peace! 🕊

### CODE REVIEW

File: site/ru/blog/new-in-devtools-101/index.md [hunk 1]
```diff
@@ -5,108 +5,146 @@ authors:
...
 tags:
   - new-in-devtools
   - devtools
   - chrome-101
-draft: true
 ---
 
 *Переводы предоставлены [Alena Batitskaia](https://twitter.com/ABatickaya). Редактор — [Maxim Salnikov](https://twitter.com/webmaxru).*
 
 {% include 'partials/devtools/ru/banner.md' %}
```
Comment (jecfish) [inline site/ru/blog/new-in-devtools-101/index.md:20]:
```suggestion

{% YouTube id='u9GRAliBrM8' %}
```

**Comment by jecfish** [comment]
Ping @webmaxru for review!

File: site/ru/blog/new-in-devtools-101/index.md [hunk 2]
```diff
@@ -5,108 +5,147 @@ authors:
...
 
 <!-- - **Export as a JSON file**. Download the recording as a JSON file. -->
+- **Export as a JSON file**. Скачать запись в виде файла JSON.
 <!-- - **Export as a @puppeteer/replay script**. Download the recording as a [Puppeteer Replay](https://github.com/puppeteer/replay) script.  -->
+- **Export as a @puppeteer/replay script**. Скачать запись как скрипт для [Puppeteer Replay](https://github.com/puppeteer/replay).
 <!-- - **Export as a Puppeteer script** . Download the recording as [Puppeteer](https://pptr.dev/) script. -->
+- **Export as a Puppeteer script**. Скачать запись как скрипт для [Puppeteer](https://pptr.dev/).
 
 <!-- Consult [the documentation](/docs/devtools/recorder) to learn more about the differences between these options. -->
+Обратитесь к [документации](/docs/devtools/recorder) чтобы узнать больше о различиях между опциями.
```
Comment (jecfish) [inline site/ru/blog/new-in-devtools-101/index.md:52]:
```suggestion
Обратитесь к [документации](/docs/devtools/recorder/#export-flows) чтобы узнать больше о различиях между опциями.
```

File: site/_includes/partials/devtools/ru/whats-new.md [hunk 3]
```diff
@@ -13,6 +13,15 @@
 * [Miscellaneous highlights](/ru/blog/new-in-devtools-101/#misc)
 * [[Experimental] New timespan and snapshot mode in the Lighthouse panel](/ru/blog/new-in-devtools-101/#lighthouse) -->
 
+### Chrome 101 {: #chrome101 }
+
+ * [Импорт и экспорт записей поведения пользователя в виде JSON файла](/ru/blog/new-in-devtools-101/#recorder)
```
Comment (webmaxru) [inline site/_includes/partials/devtools/ru/whats-new.md:18]:
```suggestion
 * [Импорт и экспорт записей действий пользователя в виде JSON файла](/ru/blog/new-in-devtools-101/#recorder)
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 4]
```diff
@@ -5,108 +5,147 @@ authors:
   - jecelynyeen
 date: 2022-04-12
 updated: 2022-04-12
-description: "Import and export user flow as JSON, support hwb() colors, view cascade layers in the Styles pane and more."
+description: "Импорт и экспорт записей поведения пользователя в виде JSON файла, поддержка функции цвета hwb(), просмотр каскадных слоёв во вкладке Стили и многое другое."
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:8]:
```suggestion
description: "Импорт и экспорт записей действий пользователя в виде JSON файла, поддержка функции цвета hwb(), просмотр каскадных слоёв во вкладке Стили и многое другое."
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 5]
```diff
@@ -5,108 +5,147 @@ authors:
...
+<!-- start: translation instructions -->
+<!-- + 1. Remove the "draft: true" tag above when submitting PR -->
+<!-- + 2. Provide translations under each of the English commented original content -->
+<!-- + 3. Translate the "description" tag above -->
+<!-- + 4. Translate all the <img> alt text -->
+<!-- + 5. Update the whats-new.md file -->
+<!-- end: translation instructions -->
+
 <!-- ## Import and export recorded user flows as a JSON file  {: #recorder } -->
+## Импорт и экспорт записей поведения пользователя в виде JSON файла  {: #recorder }
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:31]:
```suggestion
## Импорт и экспорт записей действий пользователя в виде JSON файла  {: #recorder }
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 6]
```diff
@@ -5,108 +5,147 @@ authors:
...
 
 <!-- - **Export as a JSON file**. Download the recording as a JSON file. -->
+- **Export as a JSON file**. Скачать запись в виде файла JSON.
 <!-- - **Export as a @puppeteer/replay script**. Download the recording as a [Puppeteer Replay](https://github.com/puppeteer/replay) script.  -->
+- **Export as a @puppeteer/replay script**. Скачать запись как скрипт для [Puppeteer Replay](https://github.com/puppeteer/replay).
 <!-- - **Export as a Puppeteer script** . Download the recording as [Puppeteer](https://pptr.dev/) script. -->
+- **Export as a Puppeteer script**. Скачать запись как скрипт для [Puppeteer](https://pptr.dev/).
 
 <!-- Consult [the documentation](/docs/devtools/recorder) to learn more about the differences between these options. -->
+Обратитесь к [документации](/docs/devtools/recorder/#export-flows) чтобы узнать больше о различиях между опциями.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:52]:
```suggestion
Обратитесь к [документации](/docs/devtools/recorder/#export-flows), чтобы узнать больше о различиях между опциями.
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 7]
```diff
@@ -5,108 +5,147 @@ authors:
...
 
 <!-- [Cascade layers](/blog/cascade-layers/) enable more explicit control of your CSS files to prevent style-specificity conflicts. This is particularly useful for large codebases, design systems, and when managing third party styles in applications. -->
+[Каскадные слои](/blog/cascade-layers/) позволяют более точно контролировать файлы CSS для предотвращения конфликтов, связанных со специфичностью стилей. Это особенно полезно для большой кодовой базы, дизайн-систем и для управления стилями внешних приложений.
 
 <!-- In this [example](https://jec.fyi/demo/cascade-layer), there are 3 cascade layers defined: `page`, `component` and `base`. In the **Styles** pane, you can view each layer and its styles. -->
+В этом [примере](https://jec.fyi/demo/cascade-layer) определено 3 каскадных слоя: `page`, `component` и `base`. Во вкладке **Стили** (Styles) вы можете видеть каждый из этих слоёв и их стили.
 
 <!-- Click on the layer name to view the layer order. The `page` layer has the highest specificity, therefore the `box` background is green.  -->
+Кликните на имя слоя чтобы посмотреть порядок слоёв. Слой `page` имеет наибольшую специфичность, поэтому у блока `box` зелёный цвет фона.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:69]:
```suggestion
Кликните на имя слоя, чтобы посмотреть порядок слоёв. Слой `page` имеет наибольшую специфичность, поэтому у блока `box` зелёный цвет фона.
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 8]
```diff
@@ -5,108 +5,147 @@ authors:
...
 
 
 <!-- ## View cascade layers in the Styles pane {: #layer } -->
+## Просмотр каскадных слоёв во вкладке Стили (Styles) {: #layer }
 
 <!-- [Cascade layers](/blog/cascade-layers/) enable more explicit control of your CSS files to prevent style-specificity conflicts. This is particularly useful for large codebases, design systems, and when managing third party styles in applications. -->
+[Каскадные слои](/blog/cascade-layers/) позволяют более точно контролировать файлы CSS для предотвращения конфликтов, связанных со специфичностью стилей. Это особенно полезно для большой кодовой базы, дизайн-систем и для управления стилями внешних приложений.
 
 <!-- In this [example](https://jec.fyi/demo/cascade-layer), there are 3 cascade layers defined: `page`, `component` and `base`. In the **Styles** pane, you can view each layer and its styles. -->
+В этом [примере](https://jec.fyi/demo/cascade-layer) определено 3 каскадных слоя: `page`, `component` и `base`. Во вкладке **Стили** (Styles) вы можете видеть каждый из этих слоёв и их стили.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:66]:
```suggestion
В этом [примере](https://jec.fyi/demo/cascade-layer) определено 3 каскадных слоя: `page`, `component` и `base`. Во вкладке **Стили** (Styles) вы можете увидеть каждый из этих слоёв и их стили.
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 9]
```diff
@@ -5,108 +5,147 @@ authors:
...
 
 
 <!-- ## Support for the hwb() color function {: #hwb } -->
+## Поддержка функции цвета hwb() {: #hwb }
 
 <!-- You can now view and edit [HWB color format](https://drafts.csswg.org/css-color/#the-hwb-notation) in DevTools. -->
+Теперь вы можете просматривать и редактировать [цвет в формате HWB](https://drafts.csswg.org/css-color/#the-hwb-notation) в DevTools.
 
 <!-- In the **Styles** pane, hold the **Shift** key and click on any color preview to change the color format. The HWB color format is added. -->
+На вкладке **Стили** (Styles) зажмите клавишу **Shift** и кликните на превью цвета чтобы изменить формат. Добавлен формат цвета HWB.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:85]:
```suggestion
На вкладке **Стили** (Styles) зажмите клавишу **Shift** и кликните на превью цвета, чтобы изменить формат. Добавлен формат цвета HWB.
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 10]
```diff
@@ -5,108 +5,147 @@ authors:
...
 <!-- ## Miscellaneous highlights {: #misc } -->
+## Другие важные моменты {: #misc }
 
 <!-- These are some noteworthy fixes in this release: -->
+Вот несколько заслуживающих внимания исправлений в этом выпуске:
 
 <!-- - The [Back/forward cache](/blog/new-in-devtools-98/#bfcache) now displays the extension ID which blocked [bfcache](https://web.dev/bfcache/) when present.( [1284548](https://crbug.com/1284548)) -->
+- [Возвратный кэш](/blog/new-in-devtools-98/#bfcache) теперь показывает ID расширения, которое блокирует [bfcache](https://web.dev/bfcache/) при наличии. ([1284548](https://crbug.com/1284548))
 <!-- - Fixed autocompletion support for array-like objects, CSS class names, `map.get` and HTML tags. ([1297101](https://crbug.com/1297101), [1297491](https://crbug.com/1297491), [1293807](https://crbug.com/1293807), [1296983](https://crbug.com/1296983)) -->
+- Исправленна поддержка автодополнения объектов типа массив, имён CSS-классов, `map.get` и HTML-тегов. ([1297101](https://crbug.com/1297101), [1297491](https://crbug.com/1297491), [1293807](https://crbug.com/1293807), [1296983](https://crbug.com/1296983))
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:118]:
```suggestion
- Исправлена поддержка автодополнения объектов типа массив, имён CSS-классов, `map.get` и HTML-тегов. ([1297101](https://crbug.com/1297101), [1297491](https://crbug.com/1297491), [1293807](https://crbug.com/1293807), [1296983](https://crbug.com/1296983))
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 11]
```diff
@@ -5,108 +5,147 @@ authors:
...
 <!-- - Fixed incorrect highlights when double-clicking on words and undoing autocomplete. ([1298437](https://crbug.com/1298437), [1298667](https://crbug.com/1298667)) -->
+- Исправленна неправильная подсветка при двойном клике и отмене автодополнения. ([1298437](https://crbug.com/1298437), [1298667](https://crbug.com/1298667))
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:120]:
```suggestion
- Исправлена неправильная подсветка при двойном клике и отмене автодополнения. ([1298437](https://crbug.com/1298437), [1298667](https://crbug.com/1298667))
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 12]
```diff
@@ -5,108 +5,147 @@ authors:
...
 <!-- - Fixed comment keyboard shortcut in the **Sources** panel. ([1296535](https://crbug.com/1296535)) -->
+- Исправленна поддержка горячих клавиш для комментрирования во влкадке **Источники** (Sources). ([1296535](https://crbug.com/1296535))
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:122]:
```suggestion
- Исправлена поддержка горячих клавиш для комментирования во вкладке **Источники** (Sources). ([1296535](https://crbug.com/1296535))
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 13]
```diff
@@ -5,108 +5,147 @@ authors:
...
 <!-- - Re-enable support for using **Alt** (Options) key for multi selection in the **Sources** panel. ([1304070](https://crbug.com/1304070)) -->
+- Вновь включена поддержка клавиш **Alt** (Options) для множественного выбора во вкладке **Источники** (Sources). ([1304070](https://crbug.com/1304070))
 
  
 <!-- ## [Experimental] New timespan and snapshot mode in the Lighthouse panel {: #lighthouse } -->
+## [Эксперимент] Новые режимы временного промежутка и сводки во вкладке Lighthouse {: #lighthouse }
 
 {% Aside %}
 <!-- To enable the experiment, enable the **Use Lighthouse panel with timespan and snapshot modes** checkbox under **Settings** > **Experiments**. -->
+Чтобы включить эту экспериментальную функцию поставьте голочку в чекбоксе рядом с пунктом **Use Lighthouse panel with timespan and snapshot modes** в **Настройки** (Settings) > **Эксперименты** (Experiments).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:132]:
```suggestion
Чтобы включить эту экспериментальную функцию, поставьте галочку в чекбоксе рядом с пунктом **Use Lighthouse panel with timespan and snapshot modes** в **Настройки** (Settings) > **Эксперименты** (Experiments).
```

File: site/ru/blog/new-in-devtools-101/index.md [hunk 14]
```diff
@@ -5,108 +5,147 @@ authors:
...
 
 <!-- Apart from the existing **navigation** mode, the **Lighthouse** panel now support two more modes on measuring user flows - **timespan** and **snapshot**. -->
+Помимо существующего режима **навигации**, вкладка **Lighthouse** теперь поддерживает еще два режима измерения пользовательских потоков — **timespan** и **сводка**.
 
 <!-- For example, you can use the **timespan** reports to analyze user interactions. Open this [demo](https://coffee-cart.netlify.app/) page. Select the **Timespan** mode and click on **Start timespan**. On the page, click on a coffee and end the timespan. Read the report to find out the [Total Blocking Time](https://web.dev/tbt/) and [Cumulative Layout Shift](https://web.dev/cls/) that were caused by the interaction. -->
+Например, вы можете использовать отчёты в режиме **timespan** для анализа пользовательских действий. Откройте эту [демо-страницу](https://coffee-cart.netlify.app/). Выберите режим **Timespan** и нажмите **Start timespan**. Кликните на кофе на странице и закончите запись. Прочитайте отчёт чтобы узнать [Total Blocking Time](https://web.dev/tbt/) и [Cumulative Layout Shift](https://web.dev/cls/) обусловленные взаимодействием.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-101/index.md:139]:
```suggestion
Например, вы можете использовать отчёты в режиме **timespan** для анализа пользовательских действий. Откройте эту [демо-страницу](https://coffee-cart.netlify.app/). Выберите режим **Timespan** и нажмите **Start timespan**. Кликните на кофе на странице и закончите запись. Прочитайте отчёт, чтобы узнать [Total Blocking Time](https://web.dev/tbt/) и [Cumulative Layout Shift](https://web.dev/cls/) обусловленные взаимодействием.
```

**Comment by webmaxru** [comment]
Review: completed :)

File: site/ru/blog/new-in-devtools-101/index.md [hunk 15]
```diff
@@ -5,108 +5,147 @@ authors:
...
 
 <!-- Apart from the existing **navigation** mode, the **Lighthouse** panel now support two more modes on measuring user flows - **timespan** and **snapshot**. -->
+Помимо существующего режима **навигации**, вкладка **Lighthouse** теперь поддерживает еще два режима измерения пользовательских потоков — **timespan** и **сводка**.
 
 <!-- For example, you can use the **timespan** reports to analyze user interactions. Open this [demo](https://coffee-cart.netlify.app/) page. Select the **Timespan** mode and click on **Start timespan**. On the page, click on a coffee and end the timespan. Read the report to find out the [Total Blocking Time](https://web.dev/tbt/) and [Cumulative Layout Shift](https://web.dev/cls/) that were caused by the interaction. -->
+Например, вы можете использовать отчёты в режиме **timespan** для анализа пользовательских действий. Откройте эту [демо-страницу](https://coffee-cart.netlify.app/). Выберите режим **Timespan** и нажмите **Start timespan**. Кликните на кофе на странице и закончите запись. Прочитайте отчёт, чтобы узнать [Total Blocking Time](https://web.dev/tbt/) и [Cumulative Layout Shift](https://web.dev/cls/) обусловленные взаимодействием.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-101/index.md:139]:
```suggestion
Например, вы можете использовать отчёты в режиме **timespan** для анализа пользовательских действий. Откройте эту [демо-страницу](https://coffee-cart.netlify.app/). Выберите режим **Timespan** и нажмите **Start timespan**. Кликните на кофе на странице и завершите запись. Прочитайте отчёт, чтобы узнать [Total Blocking Time](https://web.dev/tbt/) и [Cumulative Layout Shift](https://web.dev/cls/) обусловленные взаимодействием.
```

**Review by kateryna-prokopenko — COMMENTED** [review]
Thanks for the translation @solarrust, left some comments

**Comment by jecfish** [comment]
Thank you all for yet another successful translation!

**Review by jecfish — APPROVED** [review]

**Pull request merged**